applications/nrf5340_audio/src/modules/sd_card.h

```diff
@@ -60,4 +62,32 @@ int sd_card_read(char const *const filename, char *const data, size_t *size);
  */
 int sd_card_init(void);
 
+/**@brief   Open file on SD card
+ * param[in] filename   Name of file to open
+ * param[in] path_to_file Path to file
+ * @retval  0 on success
+ *              -ENODEV SD init faile. SD likely not inserted
+ *      Otherwise, error from undelying drivers
+ */
+int sd_card_segment_read_open(char const *const filename, const char *path_to_file);
```

Comment:
I would not expect either the filename nor the path_to_file to change, so:
```suggestion
int sd_card_segment_read_open(char const *const filename, char const * const path_to_file);
```